css/app.css

```diff
@@ -61,7 +61,7 @@
 }
 
 .graphiql-container .toolbar {
-  overflow-x: auto;
+  overflow-x: visible;
```

Comment:
Seems reasonable, but isn't `overflow: visible` the default? (Not sure if it's cascading down here though, haven't checked).